CMakeLists.txt

```diff
@@ -127,6 +127,7 @@ adios_option(Fortran   "Enable support for Fortran bindings" AUTO)
 adios_option(SysVShMem "Enable support for SysV Shared Memory IPC on *NIX" AUTO)
 adios_option(Profiling "Enable support for profiling" AUTO)
 adios_option(Endian_Reverse "Enable support for Little/Big Endian Interoperability" AUTO)
+adios_option(EXTERNAL_yaml-cpp "Enable support for using a external yaml-cpp library for packaging" AUTO)
```

Comment:
I think this might belong in `thirdparty/CMakeLists.txt`

Reply:
@ax3l that's a good question. At first I tried that path, but my lack of cmake knowledge showed up. I couldn't reuse the target from `find_package` coming from the thirdparty directory, it was just going away. I thought cmake target have global scope, maybe that has changed or I am missing something. In any case, we are going with @KyleFromKitware  PR #2175, so I am closing this.